rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -280,28 +280,31 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 {
 	static Timer fps_t;
 
-	++m_frames;
-
-	if (fps_t.GetElapsedTimeInSec() >= 0.5)
+	if (m_fps_title || g_cfg.misc.show_fps_in_title)
 	{
-		QString fps_title;
+		++m_frames;
 
-		if (g_cfg.misc.show_fps_in_title)
+		if (fps_t.GetElapsedTimeInSec() >= 0.5)
 		{
-			fps_title = qstr(fmt::format("FPS: %.2f", (double)m_frames / fps_t.GetElapsedTimeInSec()));
+			QString fps_title;
 
-			if (!m_windowTitle.isEmpty())
+			if ((m_fps_title = g_cfg.misc.show_fps_in_title.get()))
```

Comment:
i think the double (()) aren't needed

Reply:
It gives a warning without iirc.

Reply:
Right, Otherwise compiler complains.

Reply:
better idea maybe? :
use 
`atomic_t<bool> m_show_fps_in_title = false;`
and then use:
```
if (m_show_fps_in_title.exchange(g_cfg.misc.show_fps_in_title.get()))
    ++m_frames;
    ...
    if (g_cfg.misc.show_fps_in_title)
    {
        update fps
    }
....
```

Reply:
See new patch